deliaioanac/assignment2/LCAUnitTests.cpp

```diff
@@ -0,0 +1,71 @@
+#include "LCA.cpp"
+#include <gtest/gtest.h>
+#include <vector>
+
+namespace {
+    class LCATest : public ::testing::Test {
+        protected:
+            static CustomTree tree;
+
+            static void SetUpTestCase() {
+                /*
+                 * Constructing the following tree :
+                 * 
+                 *                1
+                 *               / \
+                 *              /   \
+                 *             2     3
+                 *            /|\   / \
+                 *           4 5 6 10  11
+                 *          / \  |
+                 *         7  8  9
+                 */
+                std::vector<int> elements = {1, 1, 2, 2, 2, 4, 4, 6, 3, 3};
+                int start = 1;
+                tree.createTree(elements, start);
+                tree.preprocessingLCA();
+            }
+
+            static void TearDownTestCase() {
+                tree.destroyTree(); 
+            }
+
+            virtual void SetUp() {}
+            virtual void TearDown() {}
```

Comment:
Correct me if I'm wrong, but I think the tests should work even if you remove lines 33-34?

Reply:
Yes, I'm sorry, I guess they were left there once I discovered the other functions.